gallery/visualtests/nxModal.js

```diff
@@ -142,6 +143,35 @@ describe('NxModal', function() {
     });
   });
 
+  describe('Nested NxModal', function() {
```

Comment:
Should probably have a visual test for this example too

Reply:
> Should probably have a visual test for this example too

Thanks, added that here: 5e819f7c58f6538c1e0290bed72c203480b9bcd0